ansible/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+hostfile = hosts.ini
+private_key_file = ~/.ssh/id_rsa
+host_key_checking = False
```

Comment:
I believe that disabling SSH host key checking by default should be avoided.  Otherwise, user can connect to a rogue host without even noticing.

If we go this route, we should at least make it crystal clear in the documentation that playbooks will not check SSH host key of targets in the inventory.